packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -2,219 +2,195 @@
-    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-hover);
-    --_iui-expandable-block-header-focus-offset: -2px;
-    --_iui-expandable-block-header-focus-thickness: 2px;
-
-    &[data-iui-expanded='true'] {
-      --_iui-expandable-block-header-bottom-border-radius: 0;
-    }
-  }
-  @if $variant == 'borderless' {
-    --_iui-expandable-block-border: none;
-    --_iui-expandable-block-background-color: transparent;
-    --_iui-expandable-block-content-padding: none;
-    --_iui-expandable-block-header-background-color-disabled: transparent;
-    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-transparent-hover);
-    --_iui-expandable-block-header-focus-offset: -1px;
-    --_iui-expandable-block-header-focus-thickness: 1px;
-
-    &[data-iui-expanded='true'] {
-      --_iui-expandable-block-header-bottom-border-radius: inherit;
-    }
-  }
-}
-
-/// @arg $size Can be one of: default, small
-@mixin iui-expandable-block-size($size: 'default') {
-  @if $size == 'default' {
-    --_iui-expandable-block-expander-icon-size: var(--iui-size-m);
-    --_iui-expandable-block-header-gap: var(--iui-size-s);
-    --_iui-expandable-block-header-padding: var(--iui-size-s);
-  }
-  @if $size == 'small' {
-    --_iui-expandable-block-expander-icon-size: var(--iui-size-s);
-    --_iui-expandable-block-header-gap: var(--iui-size-xs);
-    --_iui-expandable-block-header-padding: calc(var(--iui-size-s) * 0.5) var(--iui-size-xs);
-  }
-}
-
-@mixin iui-expandable-block-header {
-  @include mixins.iui-reset;
-  @include mixins.iui-focus(
-    $offset: var(--_iui-expandable-block-header-focus-offset),
-    $thickness: var(--_iui-expandable-block-header-focus-thickness)
-  );
-  display: flex;
-  gap: var(--_iui-expandable-block-header-gap);
-  align-items: center;
-  padding: var(--_iui-expandable-block-header-padding);
-  cursor: pointer;
-  user-select: none;
-  color: var(--iui-color-text);
-  font-family: inherit;
-  text-align: left;
-  background-color: var(--_iui-expandable-block-background-color);
-  border: var(--_iui-expandable-block-border);
-  border-radius: inherit;
-  border-bottom-left-radius: var(--_iui-expandable-block-header-bottom-border-radius, inherit);
-  border-bottom-right-radius: var(--_iui-expandable-block-header-bottom-border-radius, inherit);
-  transition: background-color var(--iui-duration-1) ease-out;
-  @media (forced-colors: active) {
-    --_iui-expandable-block-expander-icon-fill: CanvasText;
-  }
-
-  &:where(:hover) {
-    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-hover);
-    --_iui-expandable-block-header-caption-color: var(--iui-color-text-hover);
-    --_iui-expandable-block-header-label-color: var(--iui-color-text-hover);
-
-    background-color: var(--_iui-expandable-block-header-background-color-hover);
 
+  > .iui-expandable-header {
```

Comment:
almost all of the classes in this file can be _unnested_ because they have descriptive unique names now

Reply:
Removed nesting

Reply:
it's not removed though.

```scss
// original
.iui-expandable-block {
  // ...

  > .iui-expandable-header {
    // ...
  }
}
```

```scss
// expected
.iui-expandable-block {
  // ...
}
.iui-expandable-header {
  // ...
}
```

```scss
// what you did (currently)
.iui-expandable-block {
  // ...

  .iui-expandable-header {
    // ...
  }
}
```

Reply:
hmm so I moved `iui-expandable-content` and `iui-expandable-header` out (haven't pushed), but for other things like `title` and `caption` under `label`, should we move those out too? I just mainly thought it looks more organized in a hierarchal way as is
![image](https://github.com/iTwin/iTwinUI/assets/22462084/1a346a09-9577-4c1f-a3e3-1fce8172a01d)

Reply:
> for other things like title and caption under label, should we move those out too?

yes, nesting should be avoided as much as possible because it leads to hard-to-maintain code

Reply:
okay. I un-nested everything and tested functionality to make sure nothing broke :x

Reply:
for these two cases, you should be able to move the code back where it was and do nesting _opposite_ direction.
<img width="583" alt="image" src="https://github.com/iTwin/iTwinUI/assets/9084735/be29d060-05bf-469c-afe7-81e83c6e5ae7">

something like this:

```scss
.iui-expandable-content {
  // ...

  :where(.iui-expandable-block[data-iui-expanded='true']) & {
    // ...
  }

  :where(.iui-expandable-block:not([data-iui-expanded='true'])) & {
    // ...
  }
}
```

this ensures that all the code that is applied on `iui-expandable-content` is grouped together

Reply:
oh cool! I was trying to achieve something similar a while back! thanks! updated